### PR TITLE
CFP/Speakers Committee RunBook

### PR DESCRIPTION
**Pull request by jjasghar**
This is the first pass at what is expected from the CFP committee.

### CODE REVIEW

File: speakers-cfp_runbook.md [hunk 1]
```diff
@@ -1,53 +1,73 @@
...
 ### General Tasks
 
 Can you name typical tasks? For example:
 
 * Write a compelling CFP (the CFP committee)
-* Requisition a unique annual logo (swag)
-* etc.
+* Engage with the Open Source communities that we are privliged to be apart of
+* Vote and read all the CFPs that are proposed
+* Attempt to be at every meeting for the smaller and larger team, things can change quickly and CFPs can be effected
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:23]:
```suggestion
* Attempt to be at every meeting for the smaller and larger team (things can change quickly and CFPs can be affected)
```
I feel like this is a wider responsibility and not a task. Core will look into a wider responsibility list like this.

File: speakers-cfp_runbook.md [hunk 2]
```diff
@@ -1,53 +1,73 @@
...
+* Recurit or indentify possible Key Note speakers
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:24]:
```suggestion
* Recruit or identify possible keynote speakers
```

File: speakers-cfp_runbook.md [hunk 3]
```diff
@@ -1,53 +1,73 @@
...
+* Work closely with the Sponsorship and Core team to make sure that the ratio of talks to workshops to open spaces is reasonable
+* Comment or filter out "sales pitches" actively through the Open CFP process
+* Administer the CFP software for DoD ATX
+* Timely answer any and all questions for that comes in from possible or future speakers
+* Attend and Run the Office Hours, pre and post CFP for new and experianced speakers
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:29]:
```suggestion
* Attend and Run the Office Hours, pre and post CFP for new and experienced speakers
* Finalize the speaker list
* Send out acceptances
* Schedule the speakers in coordination with core
* MC the stages day of in coordination with core
```

File: speakers-cfp_runbook.md [hunk 4]
```diff
@@ -1,53 +1,73 @@
...
 
 ## Important Dates
 
-What dates are important for this role? The following headings and details are examples.
-
 ### In the beginning
 
-* Come up with good stage jokes.
+* Copy the previous years CFP, make any changes for the new theme from Core
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:35]:
```suggestion
* Copy the previous year's CFP and make any changes for the new theme from Core
```

File: speakers-cfp_runbook.md [hunk 5]
```diff
@@ -1,53 +1,73 @@
...
+* Verify that the CFP software is ready from Global Core
+* Start discussing with the Venue team to verify options if the venue has changed for workshops and breakout spaces
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:37]: I'd argue this needs to happen before the CFP is written, yes?

File: speakers-cfp_runbook.md [hunk 6]
```diff
@@ -1,53 +1,73 @@
...
 
-### 3 Months Out
+### 7-5 Months Out
 
-* Find vendors for your stage costume.
+* Prepare the CFP and publicly annouce the CFP were you can. (the specific day will require feedback from Core)
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:41]:
```suggestion
* Prepare the CFP
* Announce the CFP where you can. Coordinate with Core and Marketing.
```

File: speakers-cfp_runbook.md [hunk 7]
```diff
@@ -1,53 +1,73 @@
...
+* Recruit speakers!
+* Recruit speakers!
 
-### 2 Months Out
+### 4 months Out
 
-* Try on the costume.
-
-### 1 Month Out
+* Start a cadance of the Office Hours (once a week or some reasonable process)
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:49]:
```suggestion
* Start a cadence of the Office Hours (once a week or some reasonable process)
```

File: speakers-cfp_runbook.md [hunk 8]
```diff
@@ -1,53 +1,73 @@
...
 
-* Sell out the sponsorships.
+### 3-2 months Out
 
-### The Week Before
+* Close down the CFP
+* Start to vote and recruit others from either outside or inside the Org team to help vote on the speakers
+* Send out the speaker acceptences
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:55]:
```suggestion
* Send out the speaker acceptances
```

File: speakers-cfp_runbook.md [hunk 9]
```diff
@@ -1,53 +1,73 @@
...
+* Continue a cadance of the Office Hours (once a week or some reasonable process)
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:56]:
```suggestion
* Continue a cadence of the Office Hours (once a week or some reasonable process)
```

File: speakers-cfp_runbook.md [hunk 10]
```diff
@@ -1,53 +1,73 @@
...
 
-* Panic.
-
-### The Day Before
+### 1 Month Out
 
-* Set up.
-* Party.
+* Verify the speakers and nothing has changed last minute
+* Continue a cadance of the Office Hours (once a week or some reasonable process)
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:61]:
```suggestion
* Continue a cadence of the Office Hours (once a week or some reasonable process)
```

File: speakers-cfp_runbook.md [hunk 11]
```diff
@@ -1,53 +1,73 @@
...
-### The Week Before
+* Close down the CFP
+* Start to vote and recruit others from either outside or inside the Org team to help vote on the speakers
+* Send out the speaker acceptences
+* Continue a cadance of the Office Hours (once a week or some reasonable process)
 
-* Panic.
-
-### The Day Before
+### 1 Month Out
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:58]: Schedule?

File: speakers-cfp_runbook.md [hunk 12]
```diff
@@ -1,53 +1,73 @@
...
+* Be prepared to MC the breakout rooms
+* Be there for the speakers and what they may need
+* Remember you are the representive of DoD ATX to many of these speakers, be there and friendly!
 
 ## Reminders or Common Mistakes
 
-What do you always try to remember when you come back to this every year? Do you often find yourself scrambling for vendors? Are you always trying to remember where you left your towel? Leave notes for the next person so we can improve :)
-
+* Spend "real time" with the CFP software before opening the CFP. It can be tricky
+* You DONT have to "promote" a someone in via the CFP, you can recruit your Key Note(s) speaker(s)
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:72]:
```suggestion
* You DON'T have to "promote" someone who submitted via the CFP to a keynote slot. Recruit your keynote speaker(s) as needed.
```

File: speakers-cfp_runbook.md [hunk 13]
```diff
@@ -1,53 +1,73 @@
...
+* Gain the muscule to find "sales pitches" abstracts, but don't beat yourself up if one slips through
```
Comment (nimbinatus) [inline speakers-cfp_runbook.md:73]:
```suggestion
* Gain the muscle to find "sales pitches" abstracts, but don't beat yourself up if one slips through
```

**Review by nimbinatus — CHANGES_REQUESTED** [review]

File: speakers-cfp_runbook.md (hunk 11 above)
Comment (jjasghar) [reply speakers-cfp_runbook.md:56]: Good question, i'll have to chew on this one.

File: speakers-cfp_runbook.md (hunk 5 above)
Comment (jjasghar) [reply speakers-cfp_runbook.md:39]: My theory is that our CFP is more or less "static" for the foreseeable future. We update it with the theme and once over with anything that is obvious from last year than doesn't need to be here this year.

The amount of sessions is only really changed if we decide to axe (or add) something and I don't see us removing the three types any time soon.

File: speakers-cfp_runbook.md (hunk 11 above)
Comment (moutons) [reply speakers-cfp_runbook.md:61]: what about
* Work in partnership with the core team to build out the schedule for the event.
* Maintain communication with the speakers.
* Provide feedback on alignment with the theme and focus on attendee experience.

File: speakers-cfp_runbook.md [hunk 14]
```diff
@@ -1,53 +1,76 @@
...
+* Be prepared to MC the breakout rooms
+* Be there for the speakers and what they may need
+* Remember you are the representive of DoD ATX to many of these speakers, be there and friendly!
 
 ## Reminders or Common Mistakes
 
-What do you always try to remember when you come back to this every year? Do you often find yourself scrambling for vendors? Are you always trying to remember where you left your towel? Leave notes for the next person so we can improve :)
-
+* Spend "real time" with the CFP software before opening the CFP. It can be tricky
+* You DON'T have to "promote" someone who submitted via the CFP to a keynote slot. Recruit your keynote speaker(s) as needed.
```
Comment (moutons) [inline speakers-cfp_runbook.md:75]: really keynotes should be a core responsibility, emphasis on feedback rather than selection/recruitment here?

**Review by nimbinatus — COMMENTED** [review]
Minor nits; still waiting for a last +1

**Review by jeremytanner — APPROVED** [review]

**Review by nimbinatus — APPROVED** [review]

**Pull request merged**